scripts/generate-release-evidence.sh

```
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+set -euo pipefail
+export LC_ALL=C LANG=C
+
+ROOT_DIR=$(git -C "$(dirname "${BASH_SOURCE[0]}")/.." rev-parse --show-toplevel)
+BUILD_CONTRACT="$ROOT_DIR/configs/build/deterministic-linux-daemon.json"
+TOOL_CONTRACT="$ROOT_DIR/configs/release/tool-platform.json"
+SOURCE_REF= AMD64_DIR= ARM64_DIR= GO_SBOM_A= GO_SBOM_B= CLIENT_SBOM_A= CLIENT_SBOM_B= OUTPUT_DIR=
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)